generate/avro.go

```diff
@@ -392,6 +392,7 @@ func convertToAvroType(cfg avroConfig, expr ast.Expr, name string, nullable bool
 		default: // Another go type
 			n, err := parseGoStruct(t.Name, cfg.dir)
 			if err != nil {
+				//panic(fmt.Errorf("failed to parse Go struct: %v", err))
```

Comment:
Can this be deleted?

Reply:
We could.  I left it because it was a useful debugging tool one time when I was getting `"unknown"` returned unexpectedly.